src/portfolio/Helper.tsx

```diff
@@ -110,7 +111,12 @@ export function collectStocks(
         // check whether volatility or stock score is to be used as the key
         const val = volatility
           ? Math.round(p.stock.volatility * 100) / 100
-          : Math.round(p.stock.score * 100);
+          : // make sure score is between 0 and 100
```

Comment:
I would actually use if else instead of ternary here. This would make it more readable IMO and also get rid of the eslint-diable on top.